### PR TITLE
Add ASUS ROG Flow X16

### PR DESCRIPTION
**Pull request by somnathb1**

### CODE REVIEW

**Review by whot — CHANGES_REQUESTED** [review]
Thanks for the PR. Please make sure you submit the `sysinfo` tarball and add/link to it in the file here. See the [wiki](https://github.com/linuxwacom/libwacom/wiki/Tablet-Definition-Files) for details and/or [a recent commit](https://github.com/linuxwacom/libwacom/commit/dfc3c9a454ebb37cc4a613727ce8a141fe99a211).

**Comment by whot** [comment]
ping?

**Comment by somnathb1** [comment]
> ping?

Hi, sorry for the delay, I will submit it by tomorrow

File: data/elan-2fc2.tablet [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# ELAN touchscreen/pen sensor present in the ASUS ROG Flow X16 [GV601R]
+# Auto generated sysinfo file
```
Comment (whot) [inline data/elan-2fc2.tablet:2]:
```suggestion
# Auto generated sysinfo file
# sysinfo.layqUCS31O
```

**Review by whot — APPROVED** [review]

**Comment by whot** [comment]
Merged with a minor fix, thanks!

**Pull request merged**